src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionController.java

```diff
@@ -3,23 +3,27 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
```

Comment:
the wildcard import approach has advantages, but its usage can cause problems in certain situations

please turn off Wildcard imports (https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-)

Reply:
Okay, done